### DISCUSSION TITLE
Input area for tablet users

### DISCUSSION THREAD

**Discussion by nexies** [ppy/osu#22334] [Ideas]
Hello developers! 
Recently i've faced this problem: i've got trouble setting the input area for my tablet while trying to set up the settings in osu! lazer. 
Lazer has its' own built in ability to set up the input area for a tablet. Unfortunaly, these settings do not match with the options that the tablet driver (in my case - Wacom) provides. This becomes a matter of calculation the percentage (height and width) of the area used, which is unconvinient for a user and unaccurate. 
On this ground i've got an idea, that could solve this issue for a tablet user, but i need to know the answer on this question first:  Is it possible to get the actual physical surface dimentions of a tablet? (in centimeters or inches for instance)?

If so, the first step would be to make some adjustments or additions to the osu! settings - add an option to set the unput area similar as a tablet driver does - to set the upper-left pixel and bottom-right pixel (as shown on the screenshot)
![image](https://user-images.githubusercontent.com/63457210/213894507-970b7223-b856-4908-a9ab-8530bfd19758.png)

The second step would be achievable in case there is a way to get the actual physcal size of a tablet through the tablet driver: it is to save the dimentions of the input area in the user profile for later use. It might be very useful in case the user changes their tablet to a different one or uses an another pc to play osu - this way the user won't have to painfully try to adjust the settings for their feel.

I could try to implement this feature in game, but i am yet unfalimiar with the game's code. 
If this idea makes sence, please tell me if there's anything i can help you with to implement it!

**Comment by bdach** [comment]
We will be addressing this shortly as part of https://github.com/ppy/osu/issues/18884.
